### DISCUSSION TITLE
Displays don't work

### DISCUSSION THREAD

**Discussion by Artemka3007** [flybywiresim/aircraft#8842] [General]
Hello! I have a problem... and I really want to solve it, please help me! 🙏 I have Microsoft Flight simulator 2020 by Xatab, it's not Licensed version. I have downloaded fbw a320, all is good, but DISPLAYS they don't work, they are dark, and two displays have "If you're seeing this, instrument didn't load". What I can do with this? Help me,  Please...
![photo_5357498237906444917_y](https://github.com/user-attachments/assets/4f4f40f5-510b-4bb8-b1b1-3c686363bfb7)
![photo_5357498237906444918_y](https://github.com/user-attachments/assets/6a1ff74e-3563-48a4-bbb0-12819a2bdfa3)

**Comment by alepouna** [comment]
The aircraft requires a licensed simulator. We do not condone piracy and we do not provide support for outdated sim versions.
